AlexandreBonfim/meetup-api
# Setting up Travis for CI pipeline and Test Environment

👨‍🎨  @jonasaragao 
👨‍🍳 @AlexandreBonfim 

#### Description
The scope of this PR is to start setting up Travis for our `CI` pipeline; however as I am not one of the code owners, could not get it finished, since we'll need to enable this repo within Travis.

I have also created the environment for unit tests, and will soon update the `README` with the new scripts, we should setup a proper `linter` in a follow up PR, in order to run that on Travis also.

Finally, I has also been created the `PULL_REQUEST_TEMPLATE.md` file, in order to have an standard on our PRs, should work fine once it's merged.

#### Steps to reproduce
1. Pull the latest changes and checkout this branch
2. Run `adonis test` and the spec should pass.
3. Change the number within the spec to try to make it fail and run again.
4. It should also be possible to start the app now with `npm start`, instead of `adonis serve --dev`.

#### Unit Tests
Creating a simple spec for proof of concept.

#### References
None.

#### Risks
* None: Does not touch on currently functionality of the app, new `npm start` script might fail.
* Rollback: Revert tag.
